dts/arm/st/stm32f051.dtsi

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
```

Comment:
This is inconsistent with other submitted files.
Should probably be:
 `Copyright (c) 2018 Nathan Tsoi`